modules/lang-painless/src/main/resources/org/elasticsearch/painless/spi/java.util.regex.txt

```diff
@@ -58,6 +58,7 @@ class java.util.regex.Matcher {
   String replaceFirst(String)
   boolean requireEnd()
   Matcher reset()
+  # Whitelisting Matcher.reset(String) works around the regex limiting
```

Comment:
Since we don't actually whitelist this method, I think this is just a comment to note if we did allow that method it would allow escaping the regex limiting? Could you clarify the comment?

Reply:
Will do.

Reply:
Updated comment.